tests/test_models/nn/test_mlp.py

```diff
@@ -0,0 +1,96 @@
+from unittest.mock import MagicMock
+
+import numpy as np
+import pandas as pd
+import pytest
+import torch
+from torch import nn
+
+from etna.metrics import MAE
+from etna.models.nn import MLPModel
+from etna.models.nn.mlp import MLPNet
+from etna.transforms import FourierTransform
+from etna.transforms import LagTransform
+from etna.transforms import StandardScalerTransform
+
+
+@pytest.mark.parametrize("horizon", [8, 13])
+def test_mlp_model_run_weekly_overfit_with_scaler(ts_dataset_weekly_function_with_horizon, horizon):
+
+    ts_train, ts_test = ts_dataset_weekly_function_with_horizon(horizon)
+    lag = LagTransform(in_column="target", lags=list(range(horizon, horizon + 4)))
+    fourier = FourierTransform(period=7, order=3)
+    std = StandardScalerTransform(in_column="target")
+    ts_train.fit_transform([std, lag, fourier])
+
+    decoder_length = 14
+    model = MLPModel(
+        input_size=10,
+        encoder_length=0,
+        hidden_size=[10, 10, 10, 10, 10],
+        decoder_length=decoder_length,
+        trainer_params=dict(max_epochs=1000),
+    )
+    future = ts_train.make_future(decoder_length)
+    model.fit(ts_train)
+    future = model.forecast(future, horizon=horizon)
+
+    mae = MAE("macro")
+    assert mae(ts_test, future) < 0.05
+
+
+@pytest.fixture()
```

Comment:
Do we really need this fixture? I guess you can use any other from general conftest.py